pkgs/servers/sql/postgresql/ext/pgrouting.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, postgresql, perl, cmake, gcc, boost, gmp, cgal, mpfr }:
+
+stdenv.mkDerivation rec {
+  pname = "pgrouting";
+  version = "2.6.2";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ postgresql perl gcc boost gmp cgal mpfr ];
+
+  src = fetchFromGitHub {
+    owner  = "pgRouting";
+    repo   = pname;
+    rev    = "refs/tags/v${version}";
+    sha256 = "09xy5pmiwq0lxf2m8p4q5r892mfmn32vf8m75p84jnz4707z1l0j";
+  };
+
+  configurePhase = ''
+    mkdir -p build
+    cd build
+    cmake -L ../
```

Comment:
You shouldn't need to define a configure phase. cmake has one builtin that works, and adds some useful flags.

Reply:
Had no idea. It works!